test/tools.py

```diff
@@ -80,12 +82,11 @@ def _check_conda():
     # Conda can install required Python versions on demand
     _check_conda()
     HAS_CONDA = True
-except (RuntimeError, IOError) as exc:
+except (RuntimeError, IOError):
     HAS_CONDA = False
 
 
 try:
-    import virtualenv
```

Comment:
This will change the behavior of the program, we need to revert.

Reply:
This was failing with a flake8 error imported but not used